generators/spa/templates/Vue3JS/common/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Reflection;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("<%= friendlyName %>")]
+[assembly: AssemblyDescription("<%= extensionDescription %>")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("<%= companyName %>")]
+[assembly: AssemblyProduct("<%= friendlyName %> Extension for DNN")]
+[assembly: AssemblyCopyright("Copyright © <%= currentYear %> <%= namespaceRoot %>")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+
+// Setting ComVisible to false makes the types in this assembly not visible 
+// to COM components.  If you need to access a type in this assembly from 
+// COM, set the ComVisible attribute to true on that type.
+[assembly: ComVisible(false)]
+
+// The following GUID is for the ID of the typelib if this project is exposed to COM
+[assembly: Guid("<%= guid %>")]
+
+// Version information for an assembly consists of the following four values:
+//
+//      Major Version
+//      Minor Version 
+//      Build Number
+//      Revision
+//
+// You can specify all the values or you can default the Revision and Build Numbers 
+// by using the '*' as shown below:
+[assembly: AssemblyVersion("01.00.00")]
```

Comment:
@alejoroman0605 There is a token present in the generator that should be used here.

Reply:
@alejoroman0605 You ignored this comment.  CC: @ismaelTartola  
https://github.com/UpendoVentures/generator-upendodnn/blob/master/generators/spa/index.js#L121

Reply:
@WillStrohl I already updated https://github.com/UpendoVentures/generator-upendodnn/pull/280/commits/10c1a9a7c124c3e76beb87fba71ea268d3a06c63,
This is also wrong in the Vue 2 template, do I fix it? 
https://github.com/UpendoVentures/generator-upendodnn/blob/master/generators/spa/templates/VueJS/common/Properties/AssemblyInfo.cs#L33

Reply:
@alejoroman0605 No need to update the other one too.  Instead, an issue should be created to fix that later.  That already existed before this update.  Also, I've notified you and the rest of the team about the new tokens after that, but before this.  You should have been aware already.